include/base/OpenMCProblemBase.h

```diff
@@ -104,7 +104,7 @@ class OpenMCProblemBase : public ExternalProblem, public PostprocessorInterface
    * Get the number of particles used in the current Monte Carlo calculation
    * @return number of particles
    */
-  const int64_t & nParticles() const;
+  int nParticles() const;
```

Comment:
Just want to check to make sure this change is intentional as it doesn't seem relevant.

Reply:
Yes, it's intentional - I tried a different compiler on my Mac and got a warning here (I forget exactly what it was at this point, but it was something about binding a reference to an rvalue)

Reply:
Makes sense! There's also a type change here. If possible I think it would make sense to match the type that sets the number of particles per batch in OpenMC -- `int64_t`.

Reply:
There was actually a compiler error here, that MOOSE's templated `getParam` function doesn't like `int64_t`

Reply:
Oh strange. Is that something we should report?

Reply:
We could, but I think it's compiler-dependent in some way (because I never saw that error with the compilers on Ulam). The error message was specifically referring to unsupported `long long` type, and my guess is that maybe the templated call doesn't like having two types like that?